src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -143,4 +148,35 @@ private Subscription getExistingSubscription(final SubscriptionBase subscription
                 subscriptionBase.getConsumerGroup());
     }
 
+    private final class EventTypesHolder {
```

Comment:
@v-stepanov why don't I need it? actually, I need this list! and in your code you create this list. I check for existing event types on every call of this method. 
I just gather information about non existing event types in one loop.